rust/src/segment.rs

```diff
@@ -88,7 +88,7 @@ impl SegmentBuilder {
         self
     }
 
-    pub(crate) fn create(self, view: &BinaryView) {
+    pub(crate) fn build(self, view: &BinaryView) {
```

Comment:
What's the reasoning behind this change?

Reply:
It's simply the canonical method name for builders. Seems like right now the method name matches the underlying FFI call. This is also true for `{TypeBuilder, EnumerationBuilder, StructureBuilder}::finalize()`, but that method call is public so I was unsure whether or not break it.

Reply:
Are you saying that the Rust convention for builders to have terminals named `build`?  I can't seem to find that documented with the rest of the [builder conventions](https://rust-lang.github.io/api-guidelines/type-safety.html?highlight=build#builders-enable-construction-of-complex-values-c-builder).

If there's not a solid Rustism behind the change, I'd rather leave it as it is for people porting from Python / our existing devs.

Reply:
I think my notion of that came from the [Rust Design Patterns](https://rust-unofficial.github.io/patterns/patterns/creational/builder.html) book, which I will grant, is unofficial. The example in the official guidelines you linked uses a different verb that better suits the example use-case, so I guess it's not a super concrete -ism. I'll revert the change, squash, and force push.